src/Control/Email/Email.php

```diff
@@ -316,430 +221,142 @@ private function getDefaultFrom(): string
         return $defaultFrom;
     }
 
-    /**
-     * @return string[]
-     */
-    public function getFrom()
-    {
-        return $this->getSwiftMessage()->getFrom();
-    }
-
-    /**
-     * @param string|array $address
-     * @return string|array
-     */
-    private function sanitiseAddress($address)
+    public function setBody(AbstractPart|string $body = null): static
     {
-        if (is_array($address)) {
-            return array_map('trim', $address ?? []);
+        if ($body instanceof AbstractPart) {
+            // pass to Symfony\Component\Mime\Message::setBody()
+            return parent::setBody($body);
         }
-        return trim($address ?? '');
+        // backwards compat with 4.x
+        return $this->html($body);
```

Comment:
We're not keeping this purely for backwards compatability - there are plently of cases where the HTML might already been in a string by the time it needs to be set as the email body.
```suggestion
        // Set HTML content directly.
        return $this->html($body);
```

Reply:
Updated